### ISSUE TITLE
Style Editor bottom menu inaccessible

### ISSUE THREAD

**Issue by gab3d** [opentoonz/opentoonz#3672]
I'm having this problem with the Style Editor panel hidding the bottom menu when it shouldn't.

![imagen](https://user-images.githubusercontent.com/18104091/104080137-1aa0ec00-5205-11eb-92be-dd21af7b2ee6.png)

**Comment by RodneyBaker** [comment]
I confirm this where the Style Editor panel must be made wider in order for the menu to be accessed.

**Comment by RodneyBaker** [comment]
@gab3d While adjusting the Style Editor via Preferences Additional Style Sheet (PASS) I temporarily ran into this situation where the bottom right menu couldn't be accessed.  During that test I ended up setting a minimum and maximum size for the Style Sheet panel and really liked how that was working.  I did loosen the restriction for making the panel wider in order to accommodate seeing more brushes depending on what type of brush was being used.

All this to say, **are you using any customized settings via PASS or standard QSS/Less?**
If you are then it is fairly trivial to set the minimum width so the menu will always be accessible.
If not, then... very curious why this is happening.
